src/screens/Settings.js

```diff
@@ -62,8 +75,16 @@ class Settings extends React.Component {
    */
   handleReset = () => {
```

Comment:
I don't think I like this new Reset flow. As far as I understood it's the following:

1. If it's a default wallet, then you don't remove it from the list (i.e., you always need at least one wallet)
2. If it's not a default wallet, then you remove it from the list and you go to the welcome screen (why did you change the `push` to `/` instead of `welcome`?) And you did reset a custom wallet but when you start from the beginning you are required to start a new wallet that will substitute the default one.

Am I missing something here? I feel we should discuss this UX

Reply:
By default we will go to the `Default` wallet once we reset a wallet. Then we enter one of three cases:
- If the `Default` wallet was reset we should go to `/welcome`
- if it already exists we should go to `/locked`
- if it's a Hardware wallet we should go to `/wallet_type`

This method to choose which path happens naturally when going to the root path. That's why I chose to change from the `/welcome` to `/`.

The default wallet is the wallet that uses no prefix on the keys (this will make upgrading from an older version work seamlessly) but once it is gone we can only create wallets with a prefix.I thought we should keep the `Default` wallet as always existing because it creates a better experience when resetting a wallet, the user can choose which is his default wallet and he will always get that behavior.
This could change if we rework the init of a hardware wallet though, If the `/locked` was the default start screen of all wallets we could change to whatever wallet and the experience wouldn't change.

Reply:
Closing as the UX changed greatly.